### PR TITLE
Improve login page design

### PR DESCRIPTION
**Pull request by Blaiselubuma**

### CODE REVIEW

File: client/src/components/Login.js [hunk 1]
```diff
@@ -40,57 +50,59 @@ export class Login extends Component {
...
+              <Button
+                size="massive"
+                fluid="true"
+                type="submit"
+                content="Login"
+                primary
+              />
             </Form>
           </Segment>
+          <Segment size="huge" secondary textAlign="center">
```
Comment (llh1) [inline client/src/components/Login.js]: The text is a bit too big, can you use the normal size as we use in the rest of the website?

File: client/src/components/Login.js [hunk 2]
```diff
@@ -40,57 +50,59 @@ export class Login extends Component {
...
-                      this.props.history.push(`/`);
-                    }}
-                  />
-                </Grid.Column>
-              </Grid>
+              {loginErr ? (
+                <p>Something went wrong! Please check email or password!</p>
+              ) : null}
+              <Button
+                size="massive"
```
Comment (llh1) [inline client/src/components/Login.js]: The button is too big, let's use the regular size for the text of the button.

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/Login.js [hunk 3]
```diff
@@ -85,15 +85,15 @@ export class Login extends Component {
                 <p>Something went wrong! Please check email or password!</p>
               ) : null}
               <Button
-                size="massive"
+                size="huge"
```
Comment (llh1) [inline client/src/components/Login.js]: Hey Blaise, sorry to come back to this again, I think the size can be totally removed here, to make sure the size of the text in the button is consistent with the rest. Thanks :)

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/Login.js (hunk 3 above)
Comment (Blaiselubuma) [reply client/src/components/Login.js]: Done, size removed...

**Review by llh1 — APPROVED** [review]

**Pull request merged**